src/core/metrics/init.ts

```diff
@@ -1,10 +1,9 @@
 import { currentApplicationAtom, currentUserAtom } from '~core/shared_state';
 import { appMetrics } from '~core/metrics';
-
 // FIXME: refactor like metricsInit(appId,user) without using atoms after app init implementation
 export function metricsInit() {
   const user = currentUserAtom.getState();
   const appId = currentApplicationAtom.getState();
   // metrics starts here because now we have true user id and app id
-  appMetrics.init(appId ?? '', user?.username ?? null);
+  appMetrics.init(appId ?? '', user?.email ?? null);
```

Comment:
here's .init method
`init(appId: string, userId: string | null)`
Something is wrong :)

Reply:
we agreed with BE that userId is email

Reply:
it should be changed on init function then (and in according class)
any other person reading this code would think it's a bug

Reply:
It will be changed after app init refactor and implementing currentUser as SSoT of user data